src/services/EnvironmentVariables.ts

```diff
@@ -92,12 +92,12 @@ export class EnvironmentVariables {
 				}
 
 				if (!this.CLIENT_SESSION_TOKEN_TTL || this.CLIENT_SESSION_TOKEN_TTL.trim().length === 0) {
-					this.CLIENT_SESSION_TOKEN_TTL = "604800";
-					logger.warn("CLIENT_SESSION_TOKEN_TTL env var is not set. Setting to default - 7 days.");
+					this.CLIENT_SESSION_TOKEN_TTL = "950400";
```

Comment:
could these be renamed to CLIENT_SESSION_TOKEN_TTL_SECS pls and corresponding method to reflect the same